guides/Makefile

```
@@ -24,3 +24,6 @@ linkchecker-tryer:
 	find build -type f -name index\*html | xargs linkchecker -r1 -f common/linkchecker.ini --check-extern | ./scripts/linkchecker-tryer
 
 ccutil: $(SUBDIRS)
+
+toc: html
+	F2L=upstream_filename_to_satellite_link.json ./scripts/extract_links_toc build/**/index-*.html > build/toc.json
```

I instist you should only read `index-satellite.html` because that's the only relevant guide. And please also update that in the JSON file.
```suggestion
	F2L=upstream_filename_to_satellite_link.json ./scripts/extract_links_toc build/**/index-satellite.html > build/toc.json
```